lti_tool/views.py

```diff
@@ -184,6 +184,10 @@ def dispatch(self, request, *args, **kwargs):
     def post(self, request, *args, **kwargs):
         return super().post(request, *args, **kwargs, redirect_view='lti-submission')
 
+    def get_common_objects(self):
```

Comment:
I think this method could be moved to the class `LtiSessionMixin` so that it is not necessary to duplicate it to multiple classes.